resources/web/style/heading.pcss

```diff
@@ -50,6 +50,11 @@
     margin: 16px 0 5px;
   }
 
+  /** I _think_ h7 is not technically valid HTML tag, but in the event that the nesting is deep enough we can at least bold the text */
```

Comment:
@colleenmcginnis Thanks! I hope that most of these deep levels go away when we redo the APM IA, but it's great to have a failsafe